cpp/include/graph.hpp

```diff
@@ -212,9 +215,12 @@ class GraphCompressedSparseBaseView : public GraphViewBase<VT, ET, WT> {
    * @param  number_of_vertices    The number of vertices in the graph
    * @param  number_of_edges       The number of edges in the graph
    */
-  GraphCompressedSparseBaseView(
-    ET *offsets_, VT *indices_, WT *edge_data_, VT number_of_vertices_, ET number_of_edges_)
-    : GraphViewBase<VT, ET, WT>(edge_data_, number_of_vertices_, number_of_edges_),
+  GraphCompressedSparseBaseView(edge_t *offsets_,
+                                vertex_t *indices_,
+                                weight_t *edge_data_,
+                                vertex_t number_of_vertices_,
+                                edge_t number_of_edges_)
```

Comment:
variable names with trailing "_" is for class private member variables (RAFT clang-tidy rules).

Reply:
oops.  missed that mix.